README.md

```diff
@@ -0,0 +1,32 @@
+## <u>Progressive Streaming</u>
+
+A simple application that allows the tranfer of a video file from the server to the client and upon download, has playback. Bonus addition of a transcoder script as well. Ffmpeg libav API is used for the decoding and transcoding the video file.
```

Comment:
>> tranfer

Transfer*

>> transfer of a video file from the server to the client and upon download has playback.

Though not a review for the readme itself, but rather a note on the project itself: The main motive should be to download and playback the video simultaneously. Design an event loop whereas the buffer fill event occurs, the decoder should feed it to itself. 

>> Bonus addition of a transcoder..

This can be another small section of the readme, just as another sentence would probably look neat.

>> Ffmpeg libav API is used for the decoding and transcoding the video file.

Libs used could be another section

Reply:
Sorry for the spelling error, will fix everything

Reply:
>> Though not a review for the readme itself, but rather a note on the project itself: The main motive should be to download and playback the video simultaneously. Design an event loop whereas the buffer fill event occurs, the decoder should feed it to itself.
Yes I will work on that issue #2 soon.